modules/codelite/codelite_project.lua

```diff
@@ -366,7 +366,10 @@
 	end
 
 	function m.completion(cfg)
-		_p(3, '<Completion EnableCpp11="%s" EnableCpp14="%s">', iif(cfg.flags["C++11"], "yes", "no"), iif(cfg.flags["C++14"], "yes", "no"))
+		local cpp11 = (cfg.language == 'gnu++11') or (cfg.language == 'C++11') or cfg.flags["C++11"]
```

Comment:
This should probably be a function, to allow others to add other language extensions like GNU++11. For example, VS2015 Update 3 (probably VS2017 too) allows you to specify "C++Latest", we should have support for this to translate to C++17. When C++20 starts happening, it will need to map to that, and changing it everywhere is going to be time consuming.